src/main/row.ts

```diff
@@ -338,14 +337,15 @@ export function getBasecampTodosForHelpers(row: Row): BasecampTodoRequest[] {
         const basecampDueDate: string = getBasecampDueDate(row);
 
         if(assigneeIds.length > 0) {
-            basecampTodoRequests.push(getBasecampTodoRequest(basecampTodoContent, basecampTodoDescription, 
-                assigneeIds, leadIds, false, basecampDueDate));
+            const basecampTodoRequest = getBasecampTodoRequest(basecampTodoContent, basecampTodoDescription, assigneeIds, leadIds, false, basecampDueDate);
```

Comment:
```suggestion
            const basecampTodoRequest: BasecampTodoRequest = getBasecampTodoRequest(basecampTodoContent, basecampTodoDescription, assigneeIds, leadIds, false, basecampDueDate);
```
Also must have missed from before